src/Microsoft.IdentityModel.Tokens/Base64UrlEncoder.cs

```diff
@@ -92,7 +156,7 @@ public static string Encode(byte[] inArray)
             if (inArray == null)
                 throw LogHelper.LogArgumentNullException("inArray");
 
-            return EncodeString(Convert.ToBase64String(inArray, 0, inArray.Length));
+            return Encode(inArray, 0, inArray.Length);
         }
 
         internal static string EncodeString(string str)
```

Comment:
Should we preserve this method and its sub-methods or is it safe to remove them?  It is currently unreferenced by Wilson.

Reply:
Seems if you were to check the projects listed as InternalsVisibleTo (\src\Microsoft.IdentityModel.Tokens\Properties\AssemblyInfo.cs) and none of them used it, removing would be a good thing imo.